src/Core/Manifest/ClassManifest.php

```diff
@@ -53,6 +53,19 @@ class ClassManifest
      */
     protected $cacheKey;
 
+    /**
+     * In memory cache array for individually parsed files
+     * @var array|null
+     */
+    protected ?array $filesCache = null;
+
+    /**
+     * Key to use for files cache
+     *
+     * @var string
+     */
+    protected string $filesCacheKey;
```

Comment:
```suggestion
    /**
     * In memory cache array for individually parsed files
     */
    protected ?array $filesCache = null;

    /**
     * Key to use for files cache
     */
    protected string $filesCacheKey;
```
We don't need a PHPDoc type if there's a strong typehint - it'll just end up getting out of sync.